docs/videos.rst

```diff
@@ -0,0 +1,7 @@
+Videos
+======
+
+The following links are to video tutorials and presentations.
+
+- `Dockstore YouTube channel <https://www.youtube.com/channel/UCFWNYqxQvVLAuZq8rdOSE4g>`_
+- `Dockstore Upload Tutorial - Brian O\'Connor (2016) <https://www.youtube.com/watch?v=sInP-ByF9xU>`_
```

Comment:
Title for this one is unclear, maybe just "Dockstore tutorial (2016)"
(upload tutorial sounds like we have some feature that uploads to youtube)

Reply:
How applicable is the 2016 video?  Is it still relevant?

Reply:
Taking a quick look, I think its still worth including